llvm/include/llvm/Transforms/Vectorize/LoopVectorizationLegality.h

```diff
@@ -551,6 +602,17 @@ class LoopVectorizationLegality {
   /// (potentially) make a better decision on the maximum VF and enable
   /// the use of those function variants.
   bool VecCallVariantsFound = false;
+
+  /// Indicates whether this loop has a speculative early exit, i.e. an
+  /// uncountable exiting block that is not the latch.
+  bool HasSpeculativeEarlyExit = false;
```

Comment:
For consistency, with SCEV terminology and the variables below, `Speculative` -> `Uncountable`?